src/python/WMCore/Storage/StageOutMgr.py

```diff
@@ -16,15 +16,27 @@
 import WMCore.Storage.Backends
 import WMCore.Storage.Plugins
 
-from WMCore.Services.Dashboard.DashboardAPI import stageoutPolicyReport
 from WMCore.Storage.DeleteMgr import DeleteMgr
 from WMCore.Storage.Registry import retrieveStageOutImpl
 from WMCore.Storage.StageOutError import StageOutFailure
 from WMCore.Storage.StageOutError import StageOutInitError
 from WMCore.WMException import WMException
 
 
-# If we don't import them, they cannot be ever used (bad PyCharm!)
+def stageoutPolicyReport(fileToStage, pnn, command, stageOutType, stageOutExit):
+    """
+    Prepare report about stageout policies (probably only used for the old SSB Dashboard).
```

Comment:
I see you redefine this function at place since the whole `Dashboard` Service is removed but the word ''probably' in the doc string creates the impression we have no idea if it will ever be referred from anywhere else in the code.

Reply:
Yes, the point I wanted to make here is that I do not really know whether this information is getting published to any other monitoring system (especially for analysis jobs).

Reply:
So the key word I read here is `especially for analysis jobs`. In this case I would put that clarification in the docstring as well.